### ISSUE TITLE
save-txt file interpretation

### ISSUE THREAD

**Issue by jmayank23** [ultralytics/yolov5#3478]
## ❔Question

I trained the model at image size of 512, for inference I gave an image of 4032 X 3200 pixels, that would have gotten resized to 512 (I set img 512 in detect.py as well) and the output image is the same as resolution of input image (and not 512 X 512)

My question is, what all information am I looking at in the labels text file (I used save-txt option). More specifically, are the coordinates in the txt file with respect to resized image at 512 X 512 or with respect to the input image size?

**Comment by jmayank23** [comment]
I think the data in the text file is of the form-

ClassLabel Xcenter Ycenter Width Height

Would be nice if someone could confirm

Thanks

**Comment by glenn-jocher** [comment]
@jmayank23 txt label format is described in the **Train Custom Data** tutorial:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
